vite.config.ts

```
@@ -0,0 +1,54 @@
+      dts(),
+    ],
+    resolve: {
+      alias: {
+        '@/graph': resolve(__dirname, 'src/'),
+        '@cosmograph/cosmos': resolve(__dirname, 'src/'),
+      },
+    },
+  }
+
+  if (config?.build?.lib && config.build.rollupOptions && config.build.lib.formats && config.build.lib.formats.includes('umd')) {
+    delete config.build.rollupOptions.external
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)